src/lesson_10/script.js

```diff
@@ -0,0 +1,257 @@
+const OPERABILITY = 100;
+
+function Car(startTripe, finishTripe, expenditure, engineType, engineVolume, model, year, weight, operability = OPERABILITY) {
```

Comment:
What are `startTripe`, `finishTripe` ? We do not need them 2. Only 1 Boolean value